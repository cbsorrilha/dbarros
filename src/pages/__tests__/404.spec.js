import React from "react"
import renderer from "react-test-renderer"
import NotFoundPage from '../404'


describe('Snapshot testing the NotFoundPage', () => {
    it("renders correctly", () => {
        const props = {
            data: {
                site: {
                    siteMetadata:{
                        title: "Tchurunarublazer"
                    }
                }
            },
            location: {}
        }
        const tree = renderer.create(<NotFoundPage {...props} />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
