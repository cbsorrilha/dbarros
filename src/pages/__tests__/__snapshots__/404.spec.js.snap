// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot testing the NotFoundPage renders correctly 1`] = `
Array [
  <header
    className="sc-AykKC fLWTSL"
  >
    <h1
      className="sc-AykKE jhfoAy"
    >
      <a
        className="sc-AykKD ipEYTO"
        href="/"
      >
        <img
          alt="dbarros logo"
          src="test-file-stub"
        />
      </a>
    </h1>
    <nav
      className="sc-fzXfLQ bimKsw"
    >
      <a
        href="/blog"
      >
        Blog
      </a>
      <a
        href="/"
      >
        Sobre
      </a>
    </nav>
  </header>,
  <main
    className="sc-fzXfLR bMWpZY"
  >
    <h1>
      Não encontrado
    </h1>
    <p>
      Hmmm... Acho que você não deveria estar aqui.
    </p>
  </main>,
  <footer
    className="sc-AykKF gWljpi"
  >
    <div
      className="sc-fzXfLO iqPeVM"
    >
      <pre>
        <span
          className="sc-AykKG fUcpJu"
        >
          if
        </span>
        <span
          className="sc-AykKH bhSCBp"
        >
          (
        </span>
        <span
          className="sc-AykKI iwVWIw"
        >
          true
        </span>
        <span
          className="sc-AykKH bhSCBp"
        >
          )
        </span>
        <span
          className="sc-AykKJ cteSdM"
        >
          {
        </span>
         
        <span
          className="sc-AykKK gtdKVi"
        >
          loveToCode
        </span>
        <span
          className="sc-AykKH bhSCBp"
        >
          (
        </span>
        <span
          className="sc-AykKH bhSCBp"
        >
          )
        </span>
        <span
          className="sc-AykKJ cteSdM"
        >
          ;
        </span>
         
        <span
          className="sc-AykKJ cteSdM"
        >
          }
        </span>
      </pre>
    </div>
    <div
      className="sc-fzXfLP cKvoPT"
    >
      © 
      2020
      , Built with
       
      <a
        href="https://www.gatsbyjs.org"
      >
        Gatsby
      </a>
       
      and ♥
    </div>
  </footer>,
]
`;
